packages/website-test/build.rs

```diff
@@ -13,7 +13,7 @@ struct Level {
 
 fn main() {
     let home = env::var("CARGO_MANIFEST_DIR").unwrap();
-    let pattern = format!("{}/../../website/docs/**/*.md", home);
+    let pattern = format!("{}/../../website/docs/**/*.mdx", home);
```

Comment:
looks like this is preventing a merge? Can somebody with write access do something?